freeipa/src/main/java/com/sequenceiq/freeipa/sync/FreeipaStatusInfoLogger.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.freeipa.sync;
+
+import com.sequenceiq.freeipa.entity.InstanceMetaData;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.entity.StackStatus;
+import com.sequenceiq.freeipa.service.stack.StackStatusService;
+import com.sequenceiq.freeipa.service.stack.instance.InstanceMetaDataService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class FreeipaStatusInfoLogger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FreeipaStatusInfoLogger.class);
+
+    @Inject
+    private InstanceMetaDataService instanceMetaDataService;
+
+    @Inject
+    private StackStatusService stackStatusService;
+
+    public void logFreeipaStatus(Stack stack, Set<InstanceMetaData> checkableInstances) {
+        StackStatus freeipaStatus = stackStatusService.findFirstByStackIdOrderByCreatedDesc(stack.getId());
+        String freeipaStatusInfo = String.format("freeipa stack is %s.", freeipaStatus.getStatus());
+        String allInstanceStatusInfo = createInstancesStatusInfo(checkableInstances);
+        LOGGER.debug(":::Auto sync::: freeipa status from healtch check: {} {}", freeipaStatusInfo, allInstanceStatusInfo);
+    }
+
+    private String createInstancesStatusInfo(Set<InstanceMetaData> checkableInstances) {
+        Set<InstanceMetaData> allInstanceMetaData = fetchCheckableInstancesFromDb(checkableInstances);
+        String allInstanceStatusInfo = allInstanceMetaData.stream().map(instance -> String.format("instance %s is %s", instance.getInstanceId(),
```

Comment:
it would be more readable if you break lines at stream methods, like `map` or/and `collect` instead of in the middle of `String.format`

Reply:
Thanks! Resolved.